diff --git a/dist/engine/worker-task.js b/dist/engine/worker-task.js
index 1f251f8..b360215 100644
--- a/dist/engine/worker-task.js
+++ b/dist/engine/worker-task.js
@@ -55,7 +55,7 @@ function getMessageReducer(repository) {
         return _config_1.default.rulesUnderTesting.includes(message.ruleId);
     }
     return function reducer(all, result) {
-        const messages = result.messages.filter(messageFilter);
+        const { messages } = result // Don't filter rules by ruleId, show all
         // Process only rules that are under testing
         if (messages.length === 0) {
             return all;
diff --git a/dist/file-client/results-writer.js b/dist/file-client/results-writer.js
index adb37a3..314f909 100644
--- a/dist/file-client/results-writer.js
+++ b/dist/file-client/results-writer.js
@@ -68,7 +68,7 @@ async function writeResults(messages, repository) {
     if (!_config_1.default.CI) {
         // Construct result file name, e.g. mui-org_material-ui.md
         const repositoryOwnerAndName = repository.split('/').join('_');
-        const fileName = `${repositoryOwnerAndName}${RESULT_EXTENSION}`;
+        const fileName = `${messages.length}-${repositoryOwnerAndName}${RESULT_EXTENSION}` // Add prefix with total errors
         await new Promise((resolve, reject) => {
             const stream = fs_1.default
                 .createWriteStream(`${file_constants_1.RESULTS_LOCATION}/${fileName}`, {
